docs/components.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Components
+layout: default
+nav_order: 3
+---
+
+v0 Operator-Lifecycle-Manager was built as a [mono repo](https://github.com/operator-framework/operator-lifecycle-manager), which lead to an increase in the complexity of maintaining the project as it grew over time. 
```

Comment:
Nit:
```suggestion
Operator Lifecycle Manager v0 was built as a [mono repo](https://github.com/operator-framework/operator-lifecycle-manager), which lead to an increase in the complexity of maintaining the project as it grew over time. 
```

Reply:
Kept this as Operator-Lifecycle-Manager v0 (use of dashes) since it's a single name, but if that is not correct happy to get to rid of the dashes

Reply:
I agree with @everettraven